frontend/pages/index.js

```diff
@@ -82,7 +82,7 @@ const Home = () => {
 
           {/* Start of Description */}
           <div
-            data-aos={isMobile ? "fade" : "fade-right"}
+            data-aos={isMobile ? "fade" : "fade-up"}
```

Comment:
Is there a reason why these changes are in this PR?

Reply:
Using `"fade-left"` creates extra space on the right of the screen until the animation runs, which causes the navbar to extend past the edge of the window - see the screenshot below reference.

![image](https://user-images.githubusercontent.com/101443389/188267294-b4c73fcf-25c6-45dc-aeb7-a56763089b79.png)

Reply:
Yep sounds good - feel free to leave it in then